src/webpack-system-register.js

```diff
@@ -114,7 +114,7 @@ WebpackSystemRegister.prototype.apply = function(compiler) {
 		// http://stackoverflow.com/questions/35092183/webpack-plugin-how-can-i-modify-and-re-parse-a-module-after-compilation
 		compilation.plugin('seal', () => {
 			compilation.modules.forEach(module => {
-				let isEntry = module.entry;
+				let isEntry = module.entryModule;
```

Comment:
Tried `module.entryModule` with Webpack 1.13.0, Webpack 1.14.0 (latest) and Webpack 2.2.0-rc.2.
Seems to work for all the versions.